app/scripts/directives/logViewer.js

```diff
@@ -434,16 +436,22 @@ angular.module('openshiftConsole')
                 if(!(projectName && containerName && name && url)) {
                   return;
                 }
-                $scope.$watchGroup(['context.project.metadata.name', 'options.container', 'name'], function() {
-                  angular.extend($scope, {
-                    kibanaArchiveUrl: $sce.trustAsResourceUrl(logLinks.archiveUri({
-                                        baseURL: url,
-                                        namespace: $scope.context.project.metadata.name,
-                                        namespaceUid: $scope.context.project.metadata.uid,
-                                        podname: name,
-                                        containername: $scope.options.container,
-                                        backlink: URI.encode($window.location.href)
-                                      }, $filter('annotation')($scope.context.project,'loggingDataPrefix')))
+
+                var currentUser = AuthService.UserStore().getUser().metadata.name;
+                AggregatedLoggingService.isOperationsUser(currentUser).then(function(canViewOperationsLogs) {
```

Comment:
I don't think this needs to be encapsulated this way but I defer to @spadgett.  You should be able to simply call:
`AggregatedLoggingService.isOperationsUser(currentUser)` and assign it to a var. and then pass it in as a param to `archiveURI`

Reply:
It has to be a promise since it makes a selfsubjectaccessreview request to the server, which is asynchronous